src/pynxtools_raman_multiformat/witec/witec_reader.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List, Union
+import numpy as np
+
+
+logger = logging.getLogger("pynxtools")
+
+
+def get_data_witec(self, key: str, path: str) -> Any:
```

Comment:
Not used anymore, same for `get_attr_witec`

Reply:
removed